tests/Helios.FsCheck.Tests/Channels/Sockets/Models/TcpClientSocketModel.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using Helios.Channels;
+
+namespace Helios.FsCheck.Tests.Channels.Sockets.Models
+{
+    public class TcpClientSocketModel
```

Comment:
Not used currently - had plans for it for verifying that the clients receive the echoes back from the server.